app/Services/GraphQL.php

```diff
@@ -11,7 +11,22 @@ class GraphQL
      */
     public function __construct()
     {
-        $this->client = ClientBuilder::build(config('services.graphql.url'));
+        $headers = [
+            // Identify this client for Apollo GraphQL metrics (https://bit.ly/35Vf3F1).
+            'apollographql-client-name' => 'rogue',
+        ];
+
+        // If we have an authorization token (machine clients), assign the authorization header for
+        // our GraphQL client so that we can request gated fields.
+        if (token()->exists()) {
```

Comment:
i second the support for making this an optional add!

Reply:
Yeah, this looks perf. It also makes me want a `token()->header()` method in Gateway that just gives the `Bearer ••••` bit or null for this purpose (so you don't have to worry about the conditional here)... maybe something we can add in a cleanup sprint!

Reply:
Ooh now that would be neat! I'll throw something in the ole Pivotal icebox!

Reply:
Love that idea!! that's a perfect cleanup week task, thanks for making a card @mendelB !